### ISSUE TITLE
Inspecting a volume does not provide the additional driver options

### ISSUE THREAD

**Issue by mbentley** [docker/docker#28069]
**Description**

When creating a volume using a volume driver that has options, there is no way to inspect the volume to see additional options.

**Steps to reproduce the issue:**
1. Create a volume with options:
```
$ docker volume create --driver local --opt type=nfs --opt o=addr=localhost,rw --opt device=:/data/myapp --name foo
```
2. Inspect the volume:
```
$ docker volume inspect foo
[
    {
        "Name": "foo",
        "Driver": "local",
        "Mountpoint": "/var/lib/docker/volumes/foo/_data",
        "Labels": {},
        "Scope": "local"
    }
]
```

**Describe the results you received:**
I receive the above info.

**Describe the results you expected:**
I'd expect to be able to get the addition options specified.  I have no way to inspect the volume to see that it actually is backed by a NFS server.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
$ docker version
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.7.1
 Git commit:   6f9534c
 Built:        Thu Sep  8 10:31:18 2016
 OS/Arch:      darwin/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
$ docker info
Containers: 0
 Running: 0
 Paused: 0
 Stopped: 0
Images: 79
Server Version: 1.12.1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 350
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge null host overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 4.4.20-moby
Operating System: Alpine Linux v3.4
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.953 GiB
Name: moby
ID: ALCY:J7RH:6BAJ:WTXS:ZALF:HRZC:QA4R:5E4I:TGAB:CNPP:O4U2:4NQ7
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: 17
 Goroutines: 29
 System Time: 2016-11-04T15:41:26.497104057Z
 EventsListeners: 1
No Proxy: *.local, 169.254/16
Username: mbentley
Registry: https://index.docker.io/v1/
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Docker for Mac, CS engine, OSS engine - all same behavior.

**Comment by yongtang** [comment]
@mbentley The PR #26671 has been merged into the master. Should be available in 1.13.

**Comment by mbentley** [comment]
Ah awesome, thanks!
